build.sbt

```diff
@@ -8,6 +8,8 @@ ThisBuild / organizationName := "tessellation"
 ThisBuild / evictionErrorLevel := Level.Warn
 ThisBuild / scalafixDependencies += Libraries.organizeImports
 
+ThisBuild  / envFileName := ".sbtenv"
```

Comment:
We have used `sbt-dotenv` previously but it conflicted with nix that we introduced in #138.
If you want to, you can change your task configuration to run the node from IntelliJ and it works without `sbt-dotenv` plugin, we're already using it that way.

Anyway, if you would still want to add `sbt-dotenv`, we already have the `.env` file so I'd use this one by default.

Reply:
So the main problem I've been having with IntelliJ is with the code auto-complete. I can't get sbt to work without the environment variable for my github related token. Tried several configuration options but was unable to get it working properly. I have a .sbtenv file that only contains a personal github api token that is specific to me and not checked in here. That was the easiest way to fix IJ and it doesn't appear to be breaking any unit tests. 

If you have any more details on how to fix SBT in IJ I spent a while trying to get this to work, several suggested solutions from google just didn't work and I gave up and used this plugin from a recommendation. Not sure what you mean by changing task configuration? I tried some configs for SBT but this was only way I could get it to pass. To be clear this isn't required when I do CLI stuff since SBT works fine there, it's only IntelliJ having the issue.

Reply:
You can try adding token there:

![image](https://user-images.githubusercontent.com/9077743/160611569-d4e1321b-9261-4fb3-8ccb-8897beac705a.png)


Otherwise I suggest you to put it into your `.bashrc`/`.zshrc` (depending on which shell you use). Assuming that you imported project as sbt (not BSP) then it should work.

Reply:
Its already loaded in my bash, and works totally fine on the command line again. I've tried re-importing the project and specifying sbt explicitly, and adding this path variable, but it still doesn't work. This PR is the only thing I can use to get my build environment working, and doesn't really cost very much to add since it's an optional value.

<img width="699" alt="image" src="https://user-images.githubusercontent.com/32917835/166616406-e899be93-23ca-47f8-8222-9f58f98896d7.png">